src/wallet/wallet.h

```diff
@@ -45,17 +45,18 @@ extern bool fWalletRbf;
 
 static const unsigned int DEFAULT_KEYPOOL_SIZE = 100;
 //! -paytxfee default
-static const CAmount DEFAULT_TRANSACTION_FEE = 0;
+static const CAmount DEFAULT_TRANSACTION_FEE = COIN;
 //! -fallbackfee default
-static const CAmount DEFAULT_FALLBACK_FEE = COIN;
+//mlumin: 5/2021 scaled minimum, this likely will have to change for fee reduction
+static const CAmount DEFAULT_FALLBACK_FEE = COIN/10;
 //! -mintxfee default
 static const CAmount DEFAULT_TRANSACTION_MINFEE = COIN;
 //! minimum recommended increment for BIP 125 replacement txs
-static const CAmount WALLET_INCREMENTAL_RELAY_FEE = COIN * 5;
+static const CAmount WALLET_INCREMENTAL_RELAY_FEE = COIN/10 * 5;
 //! target minimum change amount
 static const CAmount MIN_CHANGE = COIN;
 //! final minimum change amount after paying for fees
-static const CAmount MIN_FINAL_CHANGE = COIN;
+static const CAmount MIN_FINAL_CHANGE = COIN/100;
```

Comment:
I think this should stay `COIN` for now

Reply:
correct, discussed; for 1.14.4 will remain COIN, changing later